monitor/FlowMonitor.js

```diff
@@ -1028,8 +1028,11 @@ module.exports = class FlowMonitor {
     this.updateURLPart(alarmPayload, flowObj);
 
     let alarm = new Alarm.IntelAlarm(flowObj.ts, deviceIP, severity, alarmPayload);
-
-
+    if (intelObj.isOriginIPIP) {
+      alarm['p.alarm.becauseof'] = 'ip';
+    } else {
+      alarm['p.alarm.becauseof'] = 'domain';
```

Comment:
need to indicate which domain is the reason when it's domain, it could be the full domain or the sub domains until tld.

e.g. www.google.com or google.com